team13phase1.tex

```diff
@@ -67,23 +85,47 @@
   \item
     The system shall allow a property's current lease-holding guest to schedule per-door access for other current guests.
   \end{fr}
-\item
-  The system shall allow users to automatically control some aspects of a property.
+\item\label{fr:controls}
+  The system shall expose digital controls for certain aspects of its managed properties.
   \begin{fr}
-  \item\label{customizable}
-    The following aspects shall be customizable:
+  \item\label{fr:customizable}
+    The following aspects shall be controllable by lease-holding guests:
     \begin{fr}
     \item
       the hue of artificial lighting
     \item
       the brightness of artificial lighting
     \item
       the window blinds
+    \end{fr}
+  \item\label{fr:customizable:request}
+    In addition to the property aspects listed in~\ref{fr:customizable}, the system shall allow lease-holding guests to request access to the following aspects:
+    \begin{fr}
+    \item
+      the indoor temperate
+    \item
+      the dish washer
     \item
-      the indoor temperature
+      the stove
+    \item
+      the oven
     \end{fr}
-  \item\label{devices}
-    The controls shall be accessible from the following classes of devices:
+  \item\label{fr:customizable:request_approve}
+    The system shall allow a property's owner or manager to approve guests' requests as placed per~\ref{fr:customizable:request}.
+  \item\label{fr:customizable:multiple_profiles}
+    The system shall allow users to have multiple preference profiles for a single object.
+  \item
+    The system shall allow users to schedule the application of profiles from \ref{fr:customizable:multiple_profiles}.
+  \item
+    Every aspect of the system that has automation shall have physical, manual overrides.
+  \end{fr}
+\item\label{fr:software}
+  The system's software components shall be accessible over the Internet.
+  \begin{fr}
+  \item\label{fr:software:ui}
+    The system shall present a graphical user interface (GUI) to its end users.
+  \item\label{fr:software:devices}
+    The system's GUI shall support the following classes of devices:
```

Comment:
I feel like this should be under the UI section. Perhaps move the list of devices to a broader scope and just reference them from the UI section?